backend/src/main/java/de/neuefische/backend/controller/ToDoController.java

```diff
@@ -0,0 +1,65 @@
+package de.neuefische.backend.controller;
+
+import de.neuefische.backend.model.ToDo;
+import de.neuefische.backend.model.ToDoInput;
+import de.neuefische.backend.service.ToDoService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.server.ResponseStatusException;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("api")
+public class ToDoController {
+
+    ToDoService toDoService;
+
+    @Autowired
+    public ToDoController(ToDoService toDoService) {
+        this.toDoService = toDoService;
+    }
+
+    @GetMapping("all")
+    public List<ToDo> getAllToDos() {
+        if (toDoService.getAllToDos().isEmpty()) {
+            throw new ResponseStatusException(HttpStatus.NOT_FOUND);
+        }
+        return toDoService.getAllToDos().get();
+    }
+
+    @GetMapping("Description")
```

Comment:
```suggestion
    @GetMapping("description")
```
please use small letters

Reply:
why do you need this endpoint